repo_resource/common.py

```diff
@@ -105,6 +160,28 @@ def to_file(self, filename):
     def metadata(self) -> str:
         return ''
 
+    def standard(self) -> str:
+        try:
+            root = ET.fromstring(self.__version)
+            for element in root:
+                if element.tag not in TAGS:
+                    root.remove(element)
+                    print("Removed ", element.tag)
+            sorted_xml = sorted(root, key=lambda x: (
+                TAGS.index(x.tag) if x.tag in TAGS else 999,
```

Comment:
999 ? what's this number for?

Reply:
999 is the default position for unknown elements in the manifest
Comments added in code with more details